### DISCUSSION TITLE
reset time to zero

### DISCUSSION THREAD

**Discussion by Arash668** [ansys/pyaedt#2166] [Q&A] [answered]
Hi

I want to invalidate my design and save it so that the data is cleared from Ansys Maxwell. However, Maxwell2d.analyze_from_zero)() resets the time to zero and runs the simulation. I just wanted to clear the Ansys ram so that it does not crash because I am doing optimization and there are lots of parametric analysis,
So, how can I just invalidate my design?

I normally right-click on the Analysis and click reset time to zero then save the project

**Comment by DaveTwyman** [comment, marked as answer]
Hello Arash,

Likely your simulation has moved on since a year ago and this post, I'll reply anyway for completeness.

I checked and 'ResetToTimeZero' is not yet implemented in PyAEDT, there are a few functions that just haven't been added yet. For these there is the option to reach through and use the IronPython function from PyAEDT.

I've raised Issue https://github.com/ansys/pyaedt/issues/3978 to get this added.


David

**Reply by DaveTwyman** [reply]
My mistake, digging a little deeper I see "analyze_from_zero" is implemented, we were looking for something with a name similar to "Reset to Time Zero".
'analyze_from_zero' accepts the setup name as a parameter and sets only that setup to t=0s (recall we can have multiple setups in AEDT/Maxwell).
